packages/cli-kit/src/private/node/testing/ui.ts

```diff
@@ -1,10 +1,16 @@
 import {isTruthy} from '../environment/utilities.js'
 import {render} from 'ink-testing-library'
 
+/**
+ * Wait for the component to be ready to accept input.
```

Comment:
After reading the documentation I looked at the implementation expecting some "wait for stdin to be ready" and what I found is a sleep of 100 ms. Should we extend the documentation to include the rationale behind that sleep?